### PR TITLE
Standards in observation plan

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds ability to retrieve standards in observation plan.

### CODE REVIEW

File: skyportal/handlers/api/followup_request.py [hunk 1]
```diff
@@ -954,10 +958,12 @@ def observation_schedule(
...
     instrument : skyportal.models.instrument.Instrument
         The instrument that the request is made based on
-    observation_start: astropy.time.Time
+    observation_start : astropy.time.Time
         Start time for the observations
-    observation_end: astropy.time.Time
+    observation_end : astropy.time.Time
         End time for the observations
+    standards : pandas.DataFrame
+        Standard stars for inclusion in the observation plan.
```
Comment (guynir42) [inline skyportal/handlers/api/followup_request.py:966]: What needs to be the columns of this table?

File: skyportal/handlers/api/followup_request.py [hunk 2]
```diff
@@ -1276,6 +1310,38 @@ async def get(self, instrument_id):
           description: |
             Arrow-parseable date string (e.g. 2020-01-01). If provided, end time
             of observation window, otherwise 12 hours from now.
+        - in: query
+          name: includeStandards
+          nullable: true
+          required: false
```
Comment (guynir42) [inline skyportal/handlers/api/followup_request.py:1316]: Do we need nullable and required? I thought they were just opposite of each other...?

File: skyportal/handlers/api/followup_request.py [hunk 3]
```diff
@@ -1365,17 +1411,64 @@ async def get(self, instrument_id):
...
-            followup_requests = session.scalars(followup_requests).unique().all()
-            if len(followup_requests) == 0:
+                # get owned assignments
+                followup_requests = FollowupRequest.select(self.current_user).join(
+                    allocation_subquery,
+                    FollowupRequest.allocation_id == allocation_subquery.c.id,
+                )
+
+                if start_date:
+                    start_date = str(arrow.get(start_date.strip()).datetime)
```
Comment (guynir42) [inline skyportal/handlers/api/followup_request.py:1427]: Not sure if you need to cast to string, I think `created_at` is a datetime object.

File: skyportal/utils/offset.py [hunk 4]
```diff
@@ -728,22 +734,34 @@ def get_formatted_standards_list(
...
         ]
 
+    if standard_type == "ESO":
+        mag = []
+        for _, row in df.iterrows():
+            commentSplit = row["comment"].split(" ")
+            mag.append(float(commentSplit[1].replace("V=", "")))
+        df["mag"] = mag
+
+        df = df[(df["mag"] <= magnitude_range[0]) & (df["mag"] >= magnitude_range[1])]
```
Comment (guynir42) [inline skyportal/utils/offset.py:744]: Is magnitude range always going to be [faint, bright] or is it possible that the order would be [low, high]?

**Review by guynir42 — APPROVED** [review]
I have mostly minor comments.

File: skyportal/handlers/api/followup_request.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/followup_request.py:966]: good idea.

File: skyportal/handlers/api/followup_request.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/handlers/api/followup_request.py:1316]: good point.

File: skyportal/handlers/api/followup_request.py (hunk 3 above)
Comment (mcoughlin) [reply skyportal/handlers/api/followup_request.py:1427]: ok.

File: skyportal/utils/offset.py (hunk 4 above)
Comment (mcoughlin) [reply skyportal/utils/offset.py:744]: I am adding some checking to be sure they always go faint, bright

Comment (guynir42) [reply skyportal/utils/offset.py:744]: I would prefer that you just order them as you want, and let the user input them either way.

Comment (mcoughlin) [reply skyportal/utils/offset.py:744]: Ok I fixed that.

**Pull request merged**